vidaud/seta
# [seta-react] Implement documents search

## Description

This PR adds the functionality to search for documents when pressing the `Search` button on the new search page.

Additionally, it adds API response logging for development to allow monitoring the API calls without leaving the Console tab in the browser inspector.

## How to test
- Make sure you have documents indexed in ES
- Go to http://localhost/search-new or click on the `Search New [WIP]` menu link
- Type a few terms in the search box and press the `Search` button
- The list of documents should appear
- Click on the document title in the list to expand the details
  - If present, the Taxonomy list will become visible
  - If present, the matching Chunk will become visible
- Verify that match highlights are present in the results
- Verify that the API responses are properly logged in the browser's console
